.ci/travis-ci

```diff
@@ -1,9 +1,24 @@
-#!/bin/bash -e
+#!/bin/bash
+
+set -e
 
 # Fetch origin/master so that we can diff when checking coding style.
 git remote set-branches --add origin master
 git fetch
 
-docker run -v ${TRAVIS_BUILD_DIR}:/bitbox02-firmware/ \
-       -i shiftcrypto/firmware_v2:2 \
-       bash -c "make -C /bitbox02-firmware ci"
+docker run \
+	--cap-add SYS_PTRACE \
+	--volume ${TRAVIS_BUILD_DIR}:/bitbox02-firmware/ \
+	--workdir /bitbox02-firmware \
+	shiftcrypto/firmware_v2:2 \
```

Comment:
can this be in a variable so you can't forget to update the tag in the 2nd place?

Reply:
ack